src/pages/library/library.ts

```diff
@@ -12,15 +13,46 @@ import { Observable } from 'rxjs/Observable';
 })
 export class LibraryPage {
 
-  user:any
+  user:any;
+  books:any[];
 
-  constructor(private afAuth: AngularFireAuth, public navCtrl: NavController, public navParams: NavParams) {
+  constructor(
+    private afAuth: AngularFireAuth,
+    public navCtrl: NavController,
+    public navParams: NavParams,
+    private afStore: AngularFirestore,
+  ) {
     this.afAuth.authState.subscribe(user => {
-      this.user = user
+      this.user = user;
+      console.log(this.books)
     })
+  }
+
+  async getUserBooks(user): Promise<any> {
```

Comment:
This async function gets called to get all of the book documents in the books collection that are associated with that user. I think there is a better way to model this data using subcollections on each user but With pressed time I've decided to just make books a root collection and associate each book with a user